app/services/mixpanel_service.rb

```diff
@@ -18,9 +18,26 @@ def initialize
 
     @buffer = []
     @mutex = Mutex.new
-    @tracker = Mixpanel::Tracker.new(mixpanel_key) do |type, message|
-      buffer_event_for_send(type, message)
-    end
+
+    @tracker =
+      if Rails.env.production?
+        Mixpanel::Tracker.new(mixpanel_key) do |type, message|
+          buffer_event_for_send(type, message)
+        end
+      elsif Rails.env.development?
+        Struct.new("DummyTracker") do
```

Comment:
if i don't pass it any options it was yelling at me, so I had to pass in a string here